frontend/src/components/VoteAdmin/index.scss

```
@@ -0,0 +1,85 @@
+@import '~/styles/variables';
+.wrap {
+  position: relative;
+  font-size: 12px;
+  color: BLACK_000;
+  text-align: center;
+  line-height: 2.5em;
+  border-radius: 4px;
+  background-color: $GRAY_009;
+  padding: 15px;
+  width: 90%;
+  height: 21%;
+  margin-top: 60px;
+}
+.wrapTitle {
+  display: flex;
+  justify-content: space-between;
+}
+.wrapVoteInfo {
+  display: flex;
+  flex-direction: column;
+}
+.wrapVoteInfo div {
+  display: flex;
+  padding-right: 5px;
+  margin-left: 3px;
+  align-items: center;
+}
```

scss의 이점을 살려서 한번에 적용하면 어떨까요?

```suggestion
.wrapVoteInfo {
  display: flex;
  flex-direction: column;
  
  div {
    display: flex;
    padding-right: 5px;
    margin-left: 3px;
    align-items: center;
  }
}
```